app/src/main/java/com/example/voyagerx/ui/fragments/landing/list/LaunchOverviewClickListener.kt

```diff
@@ -0,0 +1,9 @@
+package com.example.voyagerx.ui.fragments.landing.list
+
+import com.example.voyagerx.ui.fragments.landing.data.LaunchOverviewData
+
+class LaunchOverviewClickListener(
+    private val listener: (launchOverviewData: LaunchOverviewData) -> Unit
+) {
+    fun onClick(launchOverviewData: LaunchOverviewData) = listener(launchOverviewData)
+}
```

Comment:
Does this need to be in the `fragments` folder? Maybe we could create a helpers folder or something.

Reply:
Yeah that sounds good to me, I wasn't too sure about where to put this. I'm used to using feature folders so I wanted to try putting everything in a single folder but a generic onClick could definitely be a helper.